### PR TITLE
Added guide to update Cassandra

### PR DESCRIPTION
**Pull request by jens-gellynck**

### CODE REVIEW

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md [hunk 1]
```diff
@@ -4,13 +4,66 @@ uid: Cassandra_updating
...
+
+1. Rename the *C:\Program Files\Cassandra* folder to *Cassandra_bak*
+
+1. Create a new folder named *Cassandra* in *C:\Program Files\*
+
+1. Copy the downloaded Cassandra binaries to this new *Cassandra* folder
+
+1. Copy the **old** *Java* folder from *C:\Program Files\Cassandra_bak\Java* to *C:\Program Files\Cassandra*
+
+1. Copy the **old** *cassandra.yaml* from *C:\Program Files\Cassandra_bak\conf\cassandra.yaml* to *C:\Program Files\Cassandra\conf*
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md:47]: Feedback TDW: Also copy jvm.options or at least the settings we tweaked

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md [hunk 2]
```diff
@@ -4,13 +4,66 @@ uid: Cassandra_updating
...
+
+1. Stop the Cassandra service
+
+1. Rename the *C:\Program Files\Cassandra* folder to *Cassandra_bak*
+
+1. Create a new folder named *Cassandra* in *C:\Program Files\*
+
+1. Copy the downloaded Cassandra binaries to this new *Cassandra* folder
+
+1. Copy the **old** *Java* folder from *C:\Program Files\Cassandra_bak\Java* to *C:\Program Files\Cassandra*
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md:45]: Feedback TDW: Also check if the log4j file should be copied

Comment (jens-gellynck) [reply user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md:45]: I checked and loglevel is by default set to INFO which should be fine

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md (hunk 1 above)
Comment (jens-gellynck) [reply user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md:47]: jvm.options from Cassandra is same as the one we deploy

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md [hunk 3]
```diff
@@ -4,13 +4,82 @@ uid: Cassandra_updating
...
+
+`[System.Environment]::SetEnvironmentVariable('CASSANDRA_HOME','C:\progra~1\Cassandra\',[System.EnvironmentVariableTarget]::Machine)`
+
+`[System.Environment]::SetEnvironmentVariable('JAVA_HOME','C:\progra~1\Cassandra\Java\',[System.EnvironmentVariableTarget]::Machine)`
+
+1. Open a PowerShell prompt (as Administrator) and execute the following command to register the Cassandra service:
+
+`cd 'C:\Program Files\Cassandra\bin\'; .\cassandra.ps1 -install`
+
+1. Now we need to make sure the location of the *Jvm* is correctly set in the registry. To do so, execute the following PowerShell command:
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md:67]: Add step to add -Dcassandra.commitlog.ignorereplayerrors=true back in registry Options

**Review by spetebroodt — APPROVED** [review]

**Pull request merged**